src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String name;
+    private String color;
+
+    public Figure() {
+
+    }
+
+    public Figure(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    abstract void draw();
```

Comment:
why do we create this method in the abstract class instead of separate interface?